### DISCUSSION TITLE
base_link

### DISCUSSION THREAD

**Discussion by peter05010402** [NVlabs/curobo#42] [Research Q&A] [answered]
Hi,
In simple_stacking.py,  Why set "panda_link0" to base_link? 
While in franka.yml, base_link="base_link"
![image](https://github.com/NVlabs/curobo/assets/42974031/ce210d0d-911a-4088-bccc-6cc952759313)

**Comment by balakumar-s** [comment, marked as answer]
They both have the same origin in the urdf so it doesn't have any effect in the behavior.
